app/assets/javascripts/projects.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function() {
+	var checkbox = document.getElementById('project_billable');
```

Comment:
We already use jQuery so please find elements with jQuery too - it helps with cross browser issues.